### DISCUSSION TITLE
What is the best way to display a geographic scalebar?

### DISCUSSION THREAD

**Discussion by kachkaev** [airbnb/visx#1140] [Ideas]
👋 folks! A newcomer here 🙂

I tried `@visx/geo` for the first time this weekend and was pretty pleased with it! It’s great that I can visualise different parts of my data without switching away from the React thinking paradigm. There is no doubt that visx is a truly remarkable library and I really look forward to see its further growth!

I have a smal question related to geo maps and wonder if anyone has any thoughts on it.

When creating maps in QGIS, it is common to place a _scalebar_ at a corner of a geographic projection. When rendering small multiples, there’s a need for only a single scale for the whole group of maps. Here are [a few examples from google images](https://www.google.co.uk/search?tbm=isch&q=geo%20scalebar) to illustrate what I mean:
 
 
<img width="1247" alt="Screenshot 2021-04-04 at 20 17 57" src="https://user-images.githubusercontent.com/608862/113519255-eed29f80-9582-11eb-8c29-1af4233a9e2f.png">

It seems possible to make a React component that would take some info on the projection and render a scale that can be rendered in the corner of an svg or on its own. Ideally, such scale would also accept props like `fitExtent` or `fitSize` in order to keep all the necessary computations hidden from the end-users (these can be pretty hard).

I’m curious if any one has got any tips or links on displaying a scale next to a visx geomap. Perhaps, there’s even scope for a new component in `@visx/geo` or `@visx/geo-scalebar`? 

**PS:** Possible inspiration source: https://github.com/HarryStevens/d3-geo-scale-bar

**Comment by williaster** [comment]
Love this idea! I haven't seen any examples of this but think a new component in `@visx/geo` would be a great addition!
